packages/node_modules/@node-red/editor-client/src/js/ui/tray.js

```diff
@@ -265,7 +265,7 @@ RED.tray = (function() {
             }
         },
         resize: handleWindowResize,
-        close: function close(done) {
+        close: function close(done, done1) {
```

Comment:
Why the extra `done` callback? This function is only `done` once - there should be no need for two callbacks.

If you need code to execute after the tray has been closed, then use the existing done callback and wrap the code in `setTimeout(..., 10)` so it will be called *after* the tidy up happens

Reply:
fixed in phase3 implementation.